### PR TITLE
Add rubric popup for consistent-evaluation

### PR DESCRIPTION
**Pull request by JamieTran**
Add d2l-consistent-evaluation-popup-rubric

### CODE REVIEW

File: rollup/rollup-wc.config.js [hunk 1]
```diff
@@ -109,6 +109,7 @@ const appFiles = [
 	'./node_modules/d2l-content-store-add-content/d2l-content-store-add-content.js',
 	'./web-components/d2l-capture-central.js',
 	'./web-components/d2l-consistent-evaluation.js',
+	'./node_modules/d2l-consistent-evaluation/d2l-consistent-evaluation-popup-rubric.js',
```
Comment (jmurray-d2l) [inline rollup/rollup-wc.config.js:112]: I would have expected `./web-components/d2l-consistent-evaluation-popup-rubric.js`? Are you certain this one is correct?

Comment (dlockhart) [reply rollup/rollup-wc.config.js:112]: We're trying to move away from using the `web-components` directory -- referencing directly form `node_modules` like this is preferred.

Comment (jmurray-d2l) [reply rollup/rollup-wc.config.js:112]: Also it looks like CI requires the component to be found, so we'll need to add it to the consistent-eval repo first

**Comment by JamieTran** [comment]
Closing until component is complete

**Comment by dlockhart** [comment]
Make sure you actually pick up the new version of consistent-eval -- you can either wait for this to happen automatically (every hour or so) or do it manually via `rm package-lock.json; rm -rf node_odules; npm install;`.

**Review by jmurray-d2l — APPROVED** [review]
4th times the charm, as they say!

**Pull request merged**